pkg/tools/create_pkg.sh

```diff
@@ -24,7 +24,13 @@ chmod +x /tmp/AmazonAgentScripts/postinstall
 rm -rf artifact
 mkdir artifact
 sudo pkgbuild --root /tmp/AmazonCWAgentPackage/ --install-location "/" --scripts /tmp/AmazonAgentScripts --identifier com.amazon.cloudwatch.agent --version=$AGENT_VERSION artifact/amazon-cloudwatch-agent.pkg
-aws s3 cp ./artifact/amazon-cloudwatch-agent.pkg "s3://$1/integration-test/packaging/$2/$3/amazon-cloudwatch-agent.pkg"
+bucketPath="s3://$1/integration-test/packaging/$2/$3/amazon-cloudwatch-agent.pkg"
+if [[ $2 = "" ]]; then
+    # SHA parameter is empty, this must be a prod, nonprod, or nightly build.
+    bucketPath="s3://$1/integration-test/packaging//$3/amazon-cloudwatch-agent.pkg"
```

Comment:
this should not have "integration-test/packaging" in the path

Reply:
fixed